plugin-utilities-test/src/test/kotlin/org/jetbrains/research/pluginUtilities/util/SimpleParametrizedTest.kt

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.research.pluginUtilities.util
+
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import java.io.File
+
+@RunWith(Parameterized::class)
+class SimpleParametrizedTestTest : ParametrizedBaseTest(getResourcesRootPath(::SimpleParametrizedTestTest)) {
+
+    @JvmField
+    @Parameterized.Parameter(0)
+    var inFile: File? = null
+
+    @JvmField
+    @Parameterized.Parameter(1)
+    var outFile: File? = null
+
+    companion object {
+        @JvmStatic
+        @Parameterized.Parameters(name = "{index}: ({0}, {1})")
```

Comment:
Can `ParametrizedBaseTest` be used in a situation where there's no `outFile`? For example, when we test if project opens we only have the `inFile` -- path to project directory. We open the project and calculate our metric

Reply:
Yes, you can

Reply:
I added an example for this case